Makefile

```diff
@@ -45,8 +52,12 @@ test: all
 install: all
 	mkdir -p $(DESTDIR)$(BINDIR)
 	cp $(BIN) $(DESTDIR)$(BINDIR)
-	ln -sf fspec-archive $(DESTDIR)$(BINDIR)/fspec-tar
-	ln -sf fspec-archive $(DESTDIR)$(BINDIR)/fspec-cpio
+	cd $(DESTDIR)$(BINDIR) \
+	  && ln -f fspec-{archive,tar} \
```

Comment:
This brace expansion syntax is non-POSIX. I'd just  write out the full commands.